supervisionorders.md

```diff
@@ -0,0 +1,97 @@
+# Supervision Orders Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint is used to create, handle and find supervision orders.
+
+## Main Endpoint
+**/api/core/supervisionorders**
+
+Provides paginated information about all supervision orders defined in the system. See next paragraph for details about how supervision orders information is displayed.
+
+## Single Supervision Order
+**/api/core/supervisionorders/<:id>**
+
+Provides information about a single supervision order defined into the system
+
+```json
+{
+  "id": 42,
+  "_links": {
+    "item": {
+      "href": "https://example.com/server/api/core/items/092b59e8-8159-4e70-98b5-93ec60bd3431"
+    },
+    "group": {
+      "href": "https://example.com/server/api/eperson/groups/4ebb837c-c2ae-4928-9bb1-6f51df4eeb60"
+    }
+  }
+```
+Supervision order properties:
+* id: the identifier assigned to the supervision order
+
+Exposed links:
+* item: the item on which this supervision order is defined
+* group: the group that can supervise the item
+
+Status codes:
+* 200 OK - if the item is found and it is visible to the current user or the anonymous user. Withdrawn items are returned
+* 401 Unauthorized - if you are not authenticated.
+* 403 Forbidden - if you are not logged in with sufficient permissions (Administrators).
+* 404 Not found - if the supervision order doesn't exist
+
+
+## Search methods
+
+### Find by Item
+**/api/core/supervisionorders/search/byItem?>uuid=<:item-uuid1>**
```

Comment:
there is an extra > after byItem
Bettere remove the 1 in the :item-uuid as it would lead to questions like there are more than 1?